src/styles/RenderLandingPage.less

```diff
@@ -0,0 +1,870 @@
+    font-family: 'moneta';
+    font-size: 3.5rem;
+    margin-bottom: 10%;
+}
+
+.data-container {
+    display: flex;
+    justify-content: space-evenly;
+    margin-bottom: 5%;
+}
+
+.first-data-point-container {
+    width: 433px;
+}
+
+.first-data-point-container h2 {
+    font-family: 'moneta';
+    font-size: 3rem;
+}
+
+.first-data-point-container h3 {
+    font-family: 'acumin';
+    font-size: 1.2rem;
+}
+
+.second-data-point-container {
+    width: 433px;
+}
+
+.second-data-point-container h2 {
+    font-family: 'beatrice_regular';
+    font-size: 3rem;
+}
+
+.second-data-point-container h3 {
+    font-family: 'acumin';
+    font-size: 1.2rem;
+}
+
+.third-data-point-container {
+    width: 433px;
+}
+
+.third-data-point-container h2 {
+    font-family: 'beatrice';
+    font-size: 3rem;
+}
+
+.third-data-point-container h3 {
+    font-family: 'acumin';
+    font-size: 1.2rem;
+}
+
+.back-to-top {
+    width: 7%;
+    font-size: 1.2rem;
+    text-decoration: underline;
+    cursor: pointer;
+    color: black;
+    margin: 5% auto;
+}
+
+.ant-space-align-center {
+    @media (max-width: 375px) {
+        display: flex;
+        flex-wrap: wrap;
+    }
+}
+
+.ant-btn:hover {
+    border-color: @secondary-color  !important;
+}
+
+@media @mobile-small {
+    .header {
+        height: 30vh;
+       display: block;
+       position: relative;
+    }
+    .hrf-logo-img-container{
+       height:fit-content;
+       position: absolute;
+       top: 0;
+       left: 0;
+       margin: 2% 2%;
+    }
+
+    .hrf-logo-img-container img {
+        height: 30px;
+        width: 48px;
+       
+    }
+   
+    .header-text-container h1 {
+        font-size: 1.5rem;
+        padding-top: 5%;
+    }
+
+    .header-text-container h3 {
+        font-size: .85rem;
+        width: 85%;
+        margin: 0 auto;
+    }
+
+    .graphs-section {
+        display: block;
+        width: 100%;
+    }
+
+    .grant-rates-by-office-graph-container {
+        margin-top: 20%;
+        margin-bottom: 40%;
+    }
+
+    .grant-rates-by-nationality-container {
+        margin-bottom: 40%;
+    }
+
+    .gr-nationality-img {
+        margin-bottom: 8%;
+    }
+
+    .grant-rates-over-time-container {
+        margin-bottom: 40%;
+    }
+
+    .middle-section {
+        display: flex;
+        flex-wrap: wrap;
+        width: 100%;
+        height: 75vh;
+        background-color: @secondary-color;
+        padding-top: 15%;
+        text-align: center;
+    }
+
+    .hrf-img {
+        height: 169px;
+        width: 324px;
+    }
+
+    .middle-section-text-container {
+        width: 85%;
+        text-align: center;
+        margin: 0 auto;
+       
+    }
+
+    .middle-section-text-container h3 {
+        font-size: .95rem;
+        color: @white;
+    }
+
+    .bottom-section h1 {
+        font-size: 2rem;
+    }
+
+    .data-container {
+        display: flex;
+        flex-wrap: wrap;
+        width: 100%;
+    }
+
+    .first-data-point-container {
```

Comment:
first , second, third data point classes could be written in a single statement to avoid property duplication ?